test/libsolidity/semanticTests/various/value_insane.sol

```diff
@@ -1,27 +0,0 @@
-contract helper {
-    function getBalance() public payable returns (uint256 myBalance) {
-        return address(this).balance;
-    }
-}
-
-
-contract test {
-    helper h;
-
-    constructor() payable {
-        h = new helper();
-    }
-
-    function sendAmount(uint256 amount) public returns (uint256 bal) {
-        return h.getBalance{value: amount + 3, gas: 1000}();
```

Comment:
The name of this test is so vague that it's useless. I couldn't tell what this was supposed to test just from that, but looking at the oldest version, it was for multiple applications of `.value()`:
```solidity
return h.getBalance.value(amount).gas(1000).value(amount + 3)();// overwrite value
```
This is no longer possible with the `{value: ...}` syntax so the test is obsolete and should have been removed when we deprecated the old syntax.